promql/engine.go

```diff
@@ -376,7 +386,11 @@ func (ng *Engine) exec(ctx context.Context, q *query) (model.Value, error) {
 	const env = "query execution"
 
 	evalTimer := q.stats.GetTimer(stats.TotalEvalTime).Start()
-	defer evalTimer.Stop()
+	defer func() {
+		evalTimer.Stop()
+		queryTimingStats.WithLabelValues("eval_total").
```

Comment:
Thinking more about it, I think I now correctly understand what you mean. I dropped it as it's indeed a sum of the other ones.